ctgan/synthesizers/ctgan.py

```diff
@@ -197,15 +196,12 @@ def _gumbel_softmax(logits, tau=1, hard=False, eps=1e-10, dim=-1):
         Returns:
             Sampled tensor of same shape as logits from the Gumbel-Softmax distribution.
         """
-        if version.parse(torch.__version__) < version.parse('1.2.0'):
-            for i in range(10):
-                transformed = functional.gumbel_softmax(logits, tau=tau, hard=hard,
```

Comment:
@fealho Do you know the importance of using this function? It seems like it is a legacy function according to https://pytorch.org/docs/stable/generated/torch.nn.functional.gumbel_softmax.html

Reply:
It's because of issue #73. It's been almost 3 years though, so it should be alright to drop support.